frontend/public/components/environment.jsx

```diff
@@ -508,13 +508,15 @@ export const EnvironmentPage = connect(stateToProps)(
       return <div className={classNames({'co-m-pane__body': !currentEnvVars.isCreate})}>
         {containerVars}
         { !currentEnvVars.isCreate && <div className="co-m-pane__body-group">
-          <div className="environment-buttons">
+          <div className="pf-c-form environment-buttons">
             {errorMessage && <Alert isInline className="co-alert" variant="danger" title={errorMessage} />}
             {stale && <Alert isInline className="co-alert" variant="info" title="The information on this page is no longer current.">Click Reload to update and lose edits, or Save Changes to overwrite.</Alert>}
             {success && <Alert isInline className="co-alert" variant="success" title={success} />}
-            {!readOnly &&
-            <button disabled={inProgress} type="submit" className="btn btn-primary" onClick={this.saveChanges}>Save</button>}
-            {!readOnly && <button disabled={inProgress} type="button" className="btn btn-default" onClick={this.reload}>Reload</button>}
+            <ActionGroup>
+              {!readOnly &&
```

Comment:
Nit:  move `!readOnly &&` to `<ActionGroup>` and remove from the buttons so you don't end up with an empty `<ActionGroup>`